lib/charms/ovn_charm.py

```diff
@@ -729,6 +733,41 @@ def render_nrpe(self):
             charm_nrpe, self.nrpe_check_services, current_unit)
         charm_nrpe.write()
 
+    @staticmethod
+    def get_machine_id():
+        """Get local machine ID.
+
+        :returns: machine ID
+        :rtype: bytearray
+        :raises: OSError
+        """
+        with open('/etc/machine-id', 'r') as fin:
+            return bytearray.fromhex(fin.read().rstrip())
```

Comment:
We need to hash this, ref [0]

0: https://www.freedesktop.org/software/systemd/man/machine-id.html